### ISSUE TITLE
Integrate service vehicle selection

### ISSUE THREAD

**Issue by aubergine10** [Celisuis/ServiceRestrictions#1]
As there are separate restrictions for collection vs. emptying, it would be super cool to have control over which vehicles are spawned for each.

For example, a cemetery:

* Collection: Hearse
* Emptying: Coroner van

**Comment by Celisuis** [comment]
No vehicle spawning code exists at the moment. It purely handles Transfer Requests right now. I may add in Vehicle Spawning Restrictions, but not until the Transfer Restriction code is polished and verified working.

**Comment by aubergine10** [comment]
Yup, I realise the vehicle selection stuff is scope creep but I'll add another two tickets to flesh out the idea for your consideration.
